config.py

```diff
@@ -352,3 +356,35 @@ def address_from_private_key(private_key):
     GE_DB_PORT = ''
     GE_DB_PASSWORD = ''
     GE_HTTP_PROVIDER = ''
+
+TRANSFER_LIMITS = {}
+TRANSFER_LIMITS['0.P7']	= 5000
+TRANSFER_LIMITS['0.P30']	= 10000
+TRANSFER_LIMITS['0.WD7']	= 0
+TRANSFER_LIMITS['0.WD30']	= 0
```

Comment:
hm, these are 0 but are not used anywhere. Could make sense to deprecate `no_transfer_allowed` in favour of simply setting `total_amount` to `0`... what are your thoughts @enjeyw ?

https://github.com/teamsempo/SempoBlockchain/blob/ac12780b7443e3efdea669af9214fa14266dbd38/app/server/utils/transfer_limits.py#L265

Reply:
that's the way it was originally built, but there were some weird confusing things that happened from the implicit assumption of `total amount == 0` being disabling the limit - for instance the system then didn't know if a transfer was being blocked because the user had their maximum allowable amount or because the transfer type wasn't allowed at all.
Once u start handling that edge case it stops being simple anymore and u may as well just be explicit